config/vector.spec.toml

```diff
@@ -84,6 +84,22 @@ dns_servers = ["0.0.0.0:53"]
   # * type: [string]
   include_labels = ["label_key1=label_value1", "label_key2=label_value2"]
 
+  # Setting this to `false` will disable the automatic merging of partial events.
+  #
+  # * optional
+  # * default: false
+  # * type: bool
+  no_auto_partial_merge = false
+  no_auto_partial_merge = true
+
+  # The field that indicates that the event is partial. A consequent stream of
+  # partial events along with the first non-partial event will be merged together.
+  #
+  # * optional
+  # * default: "_partial"
+  # * type: string
+  partial_event_marker = "_partial"
```

Comment:
Can be `null`. Do we have to specify it here?

Reply:
Nope, this is fine for now.